src/core/authentication/session/session.manager.ts

```diff
@@ -103,15 +103,16 @@ export class SessionManager {
       : requesterSession;
 
     if (impersonatee) {
-      const allowImpersonation = new Poll();
+      const allowImpersonation = new Polls.Poll<boolean>();
       await this.sessionHost.withSession(requesterSession, async () => {
         const event = new CanImpersonateHook(
           requesterSession,
           allowImpersonation,
         );
         await this.hooks.run(event);
       });
-      if (!(allowImpersonation.plurality && !allowImpersonation.vetoed)) {
+      const res = allowImpersonation.close();
+      if (!(res.winner?.choice && !res.vetoed)) {
```

Comment:
Maybe comment on this condition?  It's a little obtuse.

Reply:
https://github.com/SeedCompany/cord-api-v3/blob/a1ca1865ea46652b59734ef0d945c83be4617866/src/components/authorization/handler/can-impersonate-via-privileges.handler.ts#L15-L15
This second arg is the `choice`.
https://github.com/SeedCompany/cord-api-v3/blob/a1ca1865ea46652b59734ef0d945c83be4617866/src/common/poll.ts#L164-L164

But I get that the word `choice` isn't used until this point.

Reply:
Ok, no comment needed now that it's tweaked.  A far as the naming goes, in this case `choice` seems to represent the result of the choice made.  So what about `result` or `outcome`? `result` is kind of loaded.